code/RecaptchaField.php

```diff
@@ -162,7 +162,7 @@ public function validate($validator)
         if(SapphireTest::is_running_test()) {
             $request = $_REQUEST;
         } else {
-            $request = Controller::curr()->getRequest();
+            $request = Controller::curr()->getRequest()->postVars();
```

Comment:
Updated to be named $postVars.

I effectively need to update the tests and $_REQUEST variable, but as this is quite critical for the field to work, I postponed that to when I have more time at hands and not working on 4 things at the same time, while reviewing books and writing presentations.